assembl/static2/css/views/idea.scss

```diff
@@ -30,6 +33,51 @@
     }
   }
 
+  .level-1,
+  .level-2,
+  .level-3,
+  .level-4 {
+    padding-bottom: $post-space;
+    padding-left: $tree-indentation*2;
+    border-top: 1px solid $first-color-light;
+    border-left: 1px solid $first-color-light;
+
+    .box {
+      border-top: none;
+    }
+
+    &:last-child {
+      border-left: none !important;
+      padding-bottom: 0 !important;
+    }
+  }
+
+  .level-5 {
+    padding-bottom: $post-space;
+
+    &:last-child {
+      padding-bottom: 0 !important;
+    }
+  }
+
+  .level-2 {
+    width: 885px;
+  }
+
+  .level-3 {
+    width: 845px;
+  }
+
+  .level-4 {
+    width: 805px;
+  }
+
+  @for $i from 5 through 12 {
+    .level-#{$i} {
+      width: 765px;
```

Comment:
765px: is it an other arbitrary width value ?

Reply:
it's 925px - 40px by indexation and 925px is what we need to be align with the announcement content.